docker-assets/run_rails_server

```diff
@@ -10,4 +10,4 @@ KEY
 
 write_encryption_key
 
-bundle exec rake db:migrate db:seed && bundle exec rails server
+bundle exec rake db:migrate db:seed && bundle exec rails assets:precompile && bundle exec rails server
```

Comment:
instead of adding a third bundle exec here, I would suggest changing it to this:
`bundle exec rake db:migrate db:seed assets:precompile && bundle exec rails server`

It takes a long time to spin up for `bundle exec`, so doing it only twice is still the best solution.